api/ereqs_admin/static/admin/override.scss

```diff
@@ -46,3 +50,134 @@ $nav-menu-width: 200px;
   position: absolute;
   width: $nav-menu-width;
 }
+
+.app-admin_interface,
+.admin-interface {
+  background: #F0F0F0;
+}
+
+.object-tools a {
+  border-radius: 3px;
+  background: #FEFEFE;
+  color: #808080;
+  border: 1px solid #E4E4E4;
+  text-transform: none;
+}
+
+#content-main .submit-row input.default {
+  text-transform: none;
+}
+
+.app-admin_interface .form-row {
+  background: white;
+}
+
+.module {
+  background: none;
+}
+
+#content  h1 {
+  font-weight: 500;
+  color: #000;
+  font-size: 24px;
+  padding: 10px;
+}
+
+.admin-interface {
+  .inline-group .tabular thead th:last-child:not([class]):not([style]) {
+    text-align: left;
+  }
+
+  .module.wide h2 {
+    background: none;
+    padding: 1rem;
+    font-weight: 500;
+    color: #000;
+    font-size: 24px;
+  }
+
+  .inline-group thead th {
+    padding: 1rem;
+  }
+
+  td,
+  th,
+  thead th,
+  tfoot td {
+    line-height: 1.5rem;
+    padding: 1rem;
+    font-size: 1rem;
```

Comment:
Let's use our existing color, font-size, and spacing variables instead of writing one-offs.